src/standard_library/field_methods.md

```diff
@@ -0,0 +1,73 @@
+
+# Field
+
+After declaring a Field, you can use these common methods on it [^migrationNote]:
+
+## to_le_bits
+
+Transforms the field into an array of bits, Little Endian.
+
+``` rust
+fn to_le_bits(_x : Field, _bit_size: u32) -> [u1]
```

Comment:
I'd like to remove the slice notation from the docs since it may be removed in the future as it is buggy. Let's update this to
```rust
fn to_le_bits<N>(_x: Field, _bit_size: u32) -> [u1; N]
```